easybuild/easyconfigs/p/PyClone/PyClone-2020.9b2-GCCcore-10.2.0-Python-3.8.6.eb

```diff
@@ -0,0 +1,48 @@
+# easybuild easyconfig
+#
+#  John Dey <jfdey@fredhtuch.org>
+#
+#  Fred Hutchinson Cancer Research Center - Seattle Washington - US
+#
+easyblock = 'Bundle'
```

Comment:
@fizwit Why not use `PythonBundle`, then you don't need to specify `exts_defaultclass`, `sanity_check_paths` or `modextrapaths`, they're set correctly automatically when using `PythonBundle`

Reply:
thanks - much cleaner